packages/tasit-action/src/testHelpers/helpers.js

```diff
@@ -40,10 +40,21 @@ export const wait = async ms => {
   return new Promise(resolve => setTimeout(resolve, ms));
 };
 
+// Note: ethers created their own BigNumber type that encapsulates BN.js
+// Because of that, exists the need of extra parses between user and our API (see tests)
+// Should we:
+// - Try to intercept and parse ethers.BigNumber to BN.js (and vice-versa)?
+// or
+// - Exposes ethers.utils.BigNumber / bigNumberify()?
```

Comment:
I'm leaning towards exposing the ethers.js BigNumber. What functionality does theirs add over BN.js?

It's also worth looking at
https://github.com/trufflesuite/truffle/releases/tag/v5.0.2
and
https://github.com/trufflesuite/truffle/releases/tag/v5.0.0

There's some discussion in how they changed their use of BN in v5, and how it relates to what web3.js v1.0 has done too

Reply:
In general though, I think we want to embrace and build off of ethers.js's abstractions unless we strongly disagree with the choices they've made

Reply:
A) Seems that the main motivation is to expose an immutable BN [ricmoo issue comment](https://github.com/ethers-io/ethers.js/issues/58#issuecomment-341993414).

B) Nice! We still using v5-beta, I'll update truffle and see but I think that doesn't affect the way how `ethers` expose numbers.

C) OK, I agree. So, should we expose `TasitAction.Utils` with some BN utils if necessary?

Reply:
A) Makes sense.
B) Sounds good.
C) Hmm, yeah maybe because an end user would use those utils for writing their own tests? Or would we mostly use them for internal testing and a user of `tasit-action` wouldn't need them?

Reply:
I can't see now how exactly our users will use the SDK but, in my experience, handling with BN, `wei/ethers` conversion, hex data, etc. is better with some good utils functions.

Reply:
Do we want to track this as an issue or should we just keep it in mind? Up to you.

Reply:
For me, an issue isn't necessary.

Reply:
Okay, I'll resolve this then.